doc/nrf/ug_thingy53.rst

```diff
@@ -302,53 +279,151 @@ See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to
+   Other mentioned features are nice to have, but they are not mandatory.
+
+   All applications compatible with the Nordic Thingy:53 in the |NCS| also support the additional features.
+   Refer to the source code of the samples and applications for examples of implementation.
+
+.. _thingy53_app_partition_manager_config:
 
 Partition manager configuration
-  Thingy:53 compatible samples use :ref:`partition_manager` to define memory partitions.
-  To ensure that partition layout does not change between builds, the sample must use static partition layout that is the same as the one in Thingy:53-compatible samples in the |NCS|.
-  The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+===============================
+
+The samples compatible with the Nordic Thingy:53 use :ref:`partition_manager` to define memory partitions.
+To ensure that partition layout does not change between builds, the sample must use a static partition layout that is the same for all compatible samples in the |NCS|.
+See :file:`pm_static_thingy53_nrf5340_cpuapp.yml` in a compatible sample for the configuration used.
+The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
 
-  The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
-  Trying to access data on this partition results in an ARM fault.
+The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
+Trying to access data on this partition results in an ARM fault.
+
+It is also necessary to add a flash controller overlay for the application core to allow data to be loaded from the external flash memory to the network core using the application core RAM.
+This is done because the network core cannot access the external flash memory directly.
+See :file:`thingy53_nrf5340_cpuapp.overlay` in a compatible sample for how this is done.
+
+.. _thingy53_app_mcuboot_bootloader:
 
 MCUboot bootloader
-  The sample must enable MCUboot bootloader and use the same configuration of the bootloader as all of the Thingy:53-compatible samples in the |NCS|.
-  This is needed to let MCUboot use the sample as an image update.
+==================
 
-  The MCUboot bootloader supports serial recovery and a custom command to erase settings storage partition.
-  Erasing settings parition is needed to ensure that an application would not be booted with incompatible content of settings partition.
+Each sample must enable the MCUboot bootloader and use the same configuration of the bootloader as all of the samples in the |NCS| that are compatible with the Nordic Thingy:53.
+This is needed to let MCUboot use the sample as an image update.
```

Comment:
Suggest rephrasing to: Each sample must enable the MCUboot bootloader and use the same configuration of the bootloader as all the Nordic Thingy:53 compatible samples in the |NCS|.
This is ensures that the sample includes the MCUboot bootloader and that a MCUboot compatible image is generated when the sample is built.

Reply:
Applied.